src/shared/components/modlog.tsx

```diff
@@ -907,8 +907,9 @@ export class Modlog extends Component<ModlogRouteProps, ModlogState> {
             options={userSearchOptions}
             loading={loadingUserSearch}
           />
-          {!this.isoData.site_res.site_view.local_site
-            .hide_modlog_mod_names && (
+          {(this.amAdminOrMod ||
+            !this.isoData.site_res.site_view.local_site
+              .hide_modlog_mod_names) && (
```

Comment:
If I'm not mistaken, this will allow anyone who is a mod of a community to filter by mod even when looking at actions for communities they aren't a mod of.

Reply:
We should be okay, check the `this.amModOrAdmin` function.